netket/machine/jax.py

```diff
@@ -332,3 +332,95 @@ def JaxRbm(hilbert, alpha, dtype=complex):
         stax.serial(stax.Dense(alpha * hilbert.size), LogCoshLayer, SumLayer()),
         dtype=dtype,
     )
+
+
+def JaxMpsPeriodic(hilbert, bond_dim, diag=False, symperiod=-1, dtype=complex):
+    return Jax(hilbert, stax.serial(MpsPeriodicLayer(hilbert, bond_dim, diag, symperiod, dtype)), dtype=dtype)
+
+
+def MpsPeriodicLayer(hilbert, bond_dim, diag=False, symperiod=-1, dtype=complex):
+    #default standard deviation equals 1e-2
+    normal_init = jax.nn.initializers.normal()
```

Comment:
I think that as of now the tensors would be initialized to be real, this is something we need to correct at the level of machine.Jax I guess